features/other_a/languages.feature

```diff
@@ -1,5 +1,32 @@
 @works
 Feature: Languages
+
+  Scenario: Add a language and check abuse and support available
```

Comment:
Since we're testing two different things here (abuse support and regular support), this might be better as two different scenarios -- that way, if one breaks, it will be clear which one.

Also, I'd suggest a few more `When`s instead of `And`s to make it clear when we're switching from one thing to the next:
- `When I go to the languages page` on line 8 because we're changing from the background to the actual steps of adding a language;
- `When I follow "Add a Language"` on line 16 because we're changing from testing that Klingon was created to creating another new language
- `When I follow "Technical Support and Feedback"` on line 26 because we're changing from testing the Abuse page to testing the Support page